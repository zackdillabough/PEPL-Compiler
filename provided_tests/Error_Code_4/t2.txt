POLY T(X, Y) = X + Y;
POLY S(R, Y, U) = R^2 + Y^2 + U;
POLY Y = x + 5555;
START
INPUT A;
INPUT B;
INPUT C;

T(1,
2);

S(A);

Y(A, B);

1 2 3 4 5
