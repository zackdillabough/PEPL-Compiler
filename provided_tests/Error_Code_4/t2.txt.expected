Error Code 4: 12 14 